index.js

```diff
@@ -105,5 +105,73 @@ module.exports = {
     "no-console": "error",
     "json/*": "error",
     "import/no-internal-modules": "error",
+    "import/order": [
+      "error",
+      {
+        "newlines-between": "always-and-inside-groups",
+        groups: [
+          ["builtin", "external"],
+          "internal",
+          ["parent", "sibling"],
+          ["index", "object"],
+        ],
+        pathGroups: [
+          {
+            pattern: "@carforyou/**",
+            group: "external",
+            position: "after",
+          },
+          {
+            pattern: "~/lib/**",
+            group: "internal",
+            position: "after",
+          },
+          {
+            pattern: "~/components/hooks/**",
+            group: "internal",
+            position: "after",
+          },
+          {
+            pattern: "~/components/contexts/**",
+            group: "internal",
+            position: "after",
+          },
+          {
+            pattern: "~/assets/**",
```

Comment:
some of the patterns here are project-specific, while this package has been pretty generic so far. I would not want to couple project-specific knowledge to this package

one option would be to not enforce sorting for those. I personally do not care much tbh. if external imports are at the top, I'm fine

another option is to move the project-specific rules to the projects themselves. or at least move them to the right file (we expose multiple configs. one per project-type)

I'm open for a discussion here, but would rather accept some tradeoffs than tight coupling

Reply:
I totally concur that we should ideally separate / extract some of the rules and put them nearer to the respective projects. I think the only thing we have to do is move the `pathGroups` to the project specific es lint config, either in this repo or actually to the specific repos, I'll check the options and adapt this PR accordingly.

Reply:
@masone Do you think moving some of the more specific rules to the next config is sufficient? Or do the next projects differ too much and we'd have to move it to the project config (respective repo) itself?